contracts/mmr/MMRVerification.sol

```diff
@@ -36,6 +36,11 @@ contract MMRVerification {
             return false;
         }
 
+        // Cannot verify a proof that doesn't have any contents
+        if(proofItems.length == 0) {
+            return false;
+        }
```

Comment:
Is that not valid if there is only one element?

Reply:
I removed the check in https://github.com/Snowfork/verification-solidity/pull/8/commits/2ef900265e27b089ac813f3b729573f7100c882d and added a 1-leaf MMR test to the test suite